arviz/data/inference_data.py

```diff
@@ -589,6 +586,108 @@ def to_dataframe(
             (dfs,) = dfs.values()
         return dfs
 
+    def to_zarr(self, store=None):
+        """
+        Convert InferenceData to a :class:`zarr.hierarchy.group` using the same groups names
+        as in InferenceData.
+
+        Data groups ("observed_data", "constant_data", "predictions_constant_data") are
+        skipped.
+
+        Raises TypeError if no valid store is found.
+
+        Parameters
+        ----------
+        store: zarr.storage i.e MutableMapping or str, optional
+            Zarr storage class or path to desired DirectoryStore.
+
+        Returns
+        -------
+        zarr.hierarchy.group
+            A zarr hierarchy group containing the InferenceData.
+
+        References
+        ----------
+        https://zarr.readthedocs.io/
+        """
+
+        """Check zarr"""
+        try:
+            import zarr
+
+            assert version.parse(zarr.__version__) >= version.parse("2.5.0")
```

Comment:
You need to import version at top of the file

    from packaging import version